firestore.rules

```diff
@@ -0,0 +1,8 @@
+rules_version = '2';
+service cloud.firestore {
+  match /databases/{database}/documents {
+    match /{document=**} {
+      allow read, write: if true;
+    }
+  }
+}
```

Comment:
can this fille be moved to the test dir?

Reply:
Firebase configs are moved to test dir